homeassistant/components/notify/html5.py

```diff
@@ -133,17 +133,6 @@ def _load_config(filename):
     return {}
 
 
-class JSONBytesDecoder(json.JSONEncoder):
```

Comment:
when is this needed and why did only HTML5 use it? What are the side effects of it being active?

Reply:
When subscribing push notification, chrome sends json data as bytes. Python default encoder doesn't support bytes.
https://docs.python.org/3/library/json.html#json.JSONEncoder

This encoder just checks if instance is bytes, if so, decode it.
On the above link there is a similar example for Complex types.

Reply:
I don't understand how that works 🤔 so this line is returning an object that is bytes? https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/notify/html5.py#L162

As far as I know, there is no bytes type in JSON.

I also think that we should just convert that type to string instead of updating the function that is used all over Home Assistant to support it.